app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_posts_counter
```

Comment:
### [OPTIONAL]
**Hint** : Just like we have in post model, `after_save :update_comments_counter`